packages/react-notion-x/src/styles.css

```diff
@@ -1600,6 +1600,323 @@ svg.notion-page-icon {
   text-overflow: ellipsis;
 }
 
+.notion-calendar-view {
+  position: relative;
+  padding-left: 1px;
+}
+
+.notion-calendar-header {
+  position: absolute;
+  left: 0px;
+  right: 0px;
+  background: white;
+  z-index: 83;
+}
+
+.notion-calendar-header-inner {
+  display: flex;
+  height: 42px;
+  align-items: center;
+}
+
+.notion-calendar-header-inner-date {
+  font-weight: 600;
+  margin-left: 8px;
+  margin-right: 8px;
+  line-height: 1;
+  font-size: 14px;
+}
+
+.notion-calendar-header-inner-controls-prev {
+  user-select: none;
+  transition: background 20ms ease-in 0s;
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  border-radius: 3px;
+  height: 24px;
+  width: 24px;
+  padding: 0px;
+}
+
+.notion-calendar-header-inner-controls-prev:hover,
+.notion-calendar-header-inner-controls-today:hover,
+.notion-calendar-header-inner-controls-next:hover {
+  background: rgba(55, 53, 47, 0.08);
```

Comment:
Does this PR support dark mode? It's okay if it doesn't; it's already looking really solid. Just wanted to check since there seem to be lots of non-css-variable colors hardcoded.

Reply:
I see below the dark mode overrides; nevermind.